src/box/lua/config/instance_config.lua

```diff
@@ -1386,6 +1386,10 @@ return schema.new('instance_config', schema.record({
         }),
     }),
     replication = schema.record({
+        disabled = schema.scalar({
+            type = 'boolean',
+            default = false,
+        }),
```

Comment:
We already have `console.enabled`, `fiber.top.enabled`, `feedback.enabled`, `flightrec.enabled`. I think `replication.enabled` would be more consistent with other options.